www/%team/index.html.spt

```diff
@@ -18,7 +18,9 @@ is_team_owner = not user.ANON and team.owner == user.participant.username
 {% block head %}
     <link rel="payment" type="text/html" title="Give weekly gift on Gratipay"
           href="https://gratipay.com/{{ team.slug }}/" />
+    <meta name="ROBOTS" content="NOINDEX, NOFOLLOW" />
```

Comment:
So what is the final decision. Should it be `NOINDEX` or not and should pending be `NOINDEX` also?